src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
enum shouldn't be in plural form
```suggestion
public enum Color {

```